lib/web/desktop_playback.go

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright 2021 Gravitational, Inc.
```

Comment:
```suggestion
Copyright 2022 Gravitational, Inc.
```

I would only bother with new files, though.